core/tee/tee_ree_fs.c

```diff
@@ -423,6 +423,8 @@ static TEE_Result ree_fs_open_primitive(bool create, uint8_t *hash,
 			fdp->dfh.idx = -1;
 		*fh = (struct tee_file_handle *)fdp;
 	} else {
+		if (res == TEE_ERROR_SECURITY)
+			EMSG("Secure storage corruption detected");
```

Comment:
DMSG() instead to avoid printing this in production system?

Reply:
OK